src/main/java/org/jabref/gui/LibraryTab.java

```diff
@@ -910,7 +910,8 @@ public void resetChangeMonitor() {
                 taskExecutor,
                 dialogService,
                 preferencesService,
-                databaseNotificationPane));
+                databaseNotificationPane,
+                this));
```

Comment:
I think, better `org.jabref.gui.StateManager#activeTabProperty` should be used inside the code instead of passing down the tab.